lib/install/bin/circleci-auto_upgrade_tools

```diff
@@ -19,11 +20,11 @@ if bin/bundle.tools update; then
   git config user.email "circleci.bot@example.com"
 
   git commit -am "Upgrades Gemfile.tools dependencies"
-  git push https://$token@github.com/pubnative/pnd.git -f
+  git push https://$token@github.com/jetthoughts/jt_tools.git -f
```

Comment:
Could we extract this and set on install stage, because this is for some repo, not for jt_tool

Reply:
Changed